tests/getNationalDaysRoutes.test.js

```diff
@@ -0,0 +1,19 @@
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+
+chai.use(chaiHttp);
+should = chai.should();
+let server = require('../src/main.js');
+
+describe('GET /nationalDays', ()=>{
+    it('should return a list of national days', done => {
+        chai.request(server).get('/nationalDays').end((req, res) =>{
+            res.should.have.status(200);
+            Array.isArray(res.body).should.equal(true);
```

Comment:
Is there any other way you can check the type that would be more in line with the other assertions?